presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -261,6 +261,7 @@ SystemConfig::SystemConfig() {
           NUM_PROP(kHttpSrvIoEvbViolationThresholdMs, 1000),
           NUM_PROP(kMaxLocalExchangePartitionBufferSize, 65536),
           BOOL_PROP(kTextWriterEnabled, false),
+          BOOL_PROP(kCharNTypeEnabled, true),
```

Comment:
Maybe call this charNToVarcharImplicitCast. I also feel the default should be false. Ideally we would want to use an authentic charN.

@amitkdutta @czentgr

Reply:
Revised the config name and set the default value to false.